### ISSUE TITLE
Question about logits

### ISSUE THREAD

**Issue by endvroy** [thu-ml/tianshou#231]
- [ ] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [x] RL algorithm bug
    + [x] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [x] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [x] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [ ] I have mentioned version numbers, operating system and environment, where applicable:
  ```python
  import tianshou, torch, sys
  print(tianshou.__version__, torch.__version__, sys.version, sys.platform)
  ```
In the DQN implementation, a policy network is required as an input to the DQN policy constructor. This is strange because DQN should only learn the Q-networks, and the policy should be completely determined by the learned Q-values (and the epsilon). I do not understand why a policy network is required as an input, but not a Q-network? Is it a particular variant of DQN? If so, I hope the documentation can be clear on it. And what if I want to use my own architecture for the Q-networks?

**Comment by Trinkle23897** [comment]
The current implementation for DQNPolicy needs a network (self.model in this class) to generate Q values. DQN (or other policy) can accept any kinds of network architecture(even recurrent network)  once your input network follows the API here: https://tianshou.readthedocs.io/en/master/tutorials/cheatsheet.html#build-policy-network

**Comment by endvroy** [comment]
Thank you for the explanation. I think I was confused by the output name `logits` in the example code.

I think 'logits' typically stand for values after `log_softmax`. This is why I was confused and thought the networks is an actor rather than a Q-network. Maybe change the name to `values` to avoid confusion?

**Comment by Trinkle23897** [comment]
Indeed. Thanks for pointing out!

**Comment by Trinkle23897** [comment]
> I think 'logits' typically stand for values after `log_softmax`. This is why I was confused and thought the networks is an actor rather than a Q-network. Maybe change the name to `values` to avoid confusion?

I check the definition. In TensorFlow or PyTorch, the logits indicate the raw output of the model. You can see that the class under `torch.distributions` has two parameters "logits" and "probs", and the output of log_softmax is probs. ([doc](https://pytorch.org/docs/stable/distributions.html#binomial))
Also here: https://stackoverflow.com/a/43577384

**Comment by Trinkle23897** [comment]
I'll update the documentation today to point out the logits in our current implementation indicates the raw output of neural network, as suggested by @danagi and @youkaichao
